test/e2e/anvil/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ghcr.io/foundry-rs/foundry
```

Comment:
why do we need this Dockerfile? you can run this image directly with docker compose

Reply:
i'd also recommend to specify the exact version for this dependency (not just `:latest`)